RollGen.Domain/DomainDice.cs

```diff
@@ -49,7 +49,7 @@ public string ReplaceWrappedExpressions<T>(string source, string expressionOpen
                 var lenientReplacedDice = ReplaceDiceExpression(matchGroupValue, true);
                 var unevaluatedMatch = Evaluate(lenientReplacedDice);
                 var target = expressionOpen + matchGroupValue + expressionClose;
-                var replacement = BooleanOrType<T>(unevaluatedMatch).ToString();
+                var replacement = Utils.BooleanOrType<T>(unevaluatedMatch).ToString();
```

Comment:
@cidthecoatrack If the dice roll expression is something like 1d20 > 10 then it should return True or False, not 1 or 0.
Also, these functions are nice on their own if the end user wants to do conversion on their own from the `object`.